### PR TITLE
feat: change docs for token revocation

### PR DESCRIPTION
**Pull request by jfhinchcliffe**
Follows on from this work on token revocation in the Buildkite app: https://linear.app/buildkite/issue/FDN-2249/investigate-api-access-token-revocations

Specifically, once this feature is active, an org can revoke a token and the user who owns the token cannot re-add that organization back to the same token. They can create a new one.

Small changes on the Managing API Tokens page: https://bk-docs-pr-1964.onrender.com/docs/apis/managing-api-tokens

### CODE REVIEW

**Comment by jfhinchcliffe** [comment]
@jameshill some small changes to the docs about org's revoking tokens: https://bk-docs-pr-1964.onrender.com/docs/apis/managing-api-tokens

File: images/docs/apis/managing_api_tokens/token-view.png
Comment (mbelton-buildkite) [inline images/docs/apis/managing_api_tokens/token-view.png:1]: Ummm, `Destroy all Humans`? I feel like there's some pop culture I'm missing here haha

File: pages/apis/managing_api_tokens.md [hunk 1]
```diff
@@ -55,10 +55,7 @@ From the API access audit page, find the token whose access you want to remove.
 
 Click through the token you'd like to remove, then click the 'Remove Organization from Token' button.
 
-Removing access from a token will send a notification email to the token's owner.
-
->📘
-> Removing access from a token does not delete the token, however token owner cannot re-add your organization to their token's scope.
+Removing access from a token will send a notification email to the token's owner, and they will not be able to re-add your organization to the token's scope.
```
Comment (mbelton-buildkite) [inline pages/apis/managing_api_tokens.md:58]: Consider tightening:

```suggestion
Removing access from a token sends a notification email to the token's owner, who cannot re-add your organization to the token's scope.
```

File: pages/apis/managing_api_tokens.md [hunk 2]
```diff
@@ -72,7 +69,7 @@ No, you can change the scope and description of a token, or revoke it, but you c
 
 ### Can I re-add my organization to a token?
 
-Yes, the token owner can re-add the organization to the token from their _API Access Tokens_ settings page.
+If an organization has revoked a token, it cannot be re-added to the token. The token owner would have to create a new token with access to your organization.
```
Comment (mbelton-buildkite) [inline pages/apis/managing_api_tokens.md:72]: Should we provide a simple "no" upfront?

```suggestion
No. If an organization has revoked a token, it cannot be re-added to the token. The token owner would have to create a new token with access to your organization.
```

**Review by mbelton-buildkite — COMMENTED** [review]
Just a couple of nitpicks to consider.

File: images/docs/apis/managing_api_tokens/token-view.png
Comment (jfhinchcliffe) [reply images/docs/apis/managing_api_tokens/token-view.png:1]: It's a quote from Bender / Futurama. I can remove if needs be!

Comment (mbelton-buildkite) [reply images/docs/apis/managing_api_tokens/token-view.png:1]: Ahhhhhh gotcha. The previous places I've worked have had a strict policy of not having any pop culture references in the docs. I'm keen to try being a bit more lenient, but I think it needs to be something that doesn't sound too negative when someone doesn't have the context haha

File: pages/apis/managing_api_tokens.md (hunk 1 above)
Comment (jfhinchcliffe) [reply pages/apis/managing_api_tokens.md]: 👍

File: pages/apis/managing_api_tokens.md (hunk 2 above)
Comment (jfhinchcliffe) [reply pages/apis/managing_api_tokens.md]: changed

File: images/docs/apis/managing_api_tokens/token-view.png
Comment (jfhinchcliffe) [reply images/docs/apis/managing_api_tokens/token-view.png:1]: sure, I'll change it 😁

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me :)

**Pull request merged**